models/Transaction.js

```diff
@@ -0,0 +1,63 @@
+const { Schema, model } = require("mongoose");
+
+const transactionSchema = new Schema(
+  {
+    date: {
+      day: {
+        type: Number,
+        required: true,
+        min: 1,
+        max: 31,
+      },
+      month: {
+        type: Number,
+        required: true,
+        min: 0,
+        max: 11,
+      },
+      year: {
+        type: Number,
+        required: true,
+        min: 2024,
```

Comment:
czemu min 2024? :)

Reply:
Żeby nie wprowadzać wpłat z poprzednich lat. Jak nie potrzebne to usunąłem.